data/watchakorn-18k.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Watchakorn-18k",
+  "type": "personal",
+  "bio": "I'm a Python developer with a passion for creating innovative solutions to complex problems.",
+  "avatar": "https://github.com/watchakorn-18k.png",
```

Comment:
The `avatar` field is no longer required as of a recent update.
```suggestion
```

Reply:
Ok